### PR TITLE
Add warning about changed default for changelog

### PR DESCRIPTION
**Pull request by lbarcziova**
Related to packit/packit#2253
Merge after packit/packit.dev#854

RELEASE NOTES BEGIN

N/A

RELEASE NOTES END

### CODE REVIEW

File: packit_service/constants.py [hunk 1]
```diff
@@ -279,3 +279,19 @@ def from_number(number: int):
...
+
+
+CHANGED_DEFAULT_CHANGELOG = (
+    ":exclamation: **Changed default for the changelog entry generated by Packit**\n\n"
+    "We would like to bring to your attention a recent update "
+    "regarding the default changelog entry generated by Packit during release syncing. "
+    "Going forward, the default changelog entry will be generated as follows:\n"
+    "```\n- Update to version {version}`.\n```\nfollowed by a Bugzilla reference if there"
+    "is an opened bug about new upstream release.\n"
+    "This aligns with the Fedora packaging guidelines and has been agreed upon in "
```
Comment (nforro) [inline packit_service/constants.py:291]:
```suggestion
    "is an open bug about a new upstream release.\n"
    "This aligns with Fedora Packaging Guidelines and has been agreed upon in "
```

**Review by nforro — APPROVED** [review]
I'm just wondering if we could somehow make this easier in the future, I mean for example having some "announcement" constant that would be always appended and would be either empty or contain an announcement.

**Comment by lbarcziova** [comment]
@nforro +1, I was also thinking about this. I created a class for this, PTAL.

**Review by nforro — APPROVED** [review]
Nice, thanks!

**Pull request merged**